packages/devtools_extensions/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: devtools_extensions
+description: A package for building and supporting extensions for Dart DevTools.
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: '>=3.0.0'
+  flutter: '>=3.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
```

Comment:
I'm assuming we plan on exposing widgets / `ValueNotifier`s through this package?

Reply:
Yeah, this package will also include a template for extension authors to place at the root of their devtools extension app